sample_cgfs/rv32i_priv.cgf

```diff
@@ -1,21 +1,150 @@
 ecall:
   config: 
     - check ISA:=regex(.*I.*); def rvtest_mtrap_routine=True 
-  opcode: 
+  mnemonics: 
     ecall: 0
 
 ebreak:
   config: 
     - check ISA:=regex(.*I.*); def rvtest_mtrap_routine=True 
-  opcode: 
+  mnemonics: 
     ebreak: 0
+  
+misa:
+  config: 
+    - check ISA:=regex(.*I.*Zicsr.*); def rvtest_mtrap_routine=True; def rvtest_strap_routine=True
+  csr_comb:
+    misa >> (xlen-2) == 0x01 : 0
+    (misa >> 30) & 0x3 == 0x3 : 0
+    misa & 0x30 == 0x30 : 0
+    misa & 0x04 == 0x01 : 0
```

Comment:
I am not sure I understand the relevance of these coverpoints. What are these meant to test? It would help if the scenario being tested by these coverpoints are explained using yaml comment syntax. Also, the coverpoints need to account for the fact that `misa.extensions` is a warl field. Hence some of the coverpoints need not be applicable in certain scenarios.